Sources/Basic/TerminalController.swift

```diff
@@ -78,11 +78,13 @@ public final class TerminalController {
             return width
         }
 
+#if TIOCGWINSZ
```

Comment:
Is there any other way to check if variables are defined in Swift?

```
--- bootstrap: note: building stage1: /usr/bin/swift-build-tool -f /home/ruscur/swiftpm/.build/.bootstrap/build.swift-build
Compile Swift Module 'Basic' (24 sources)
Link PackageDescription
/home/ruscur/swiftpm/Sources/Basic/TerminalController.swift:83:26: error: use of unresolved identifier 'TIOCGWINSZ'
        if ioctl(1, UInt(TIOCGWINSZ), &ws) == 0 {
                         ^~~~~~~~~~
<unknown>:0: error: build had 1 command failures
--- bootstrap: error: build failed with exit status 1
```

This is on powerpc64le so I suspect something greater is going wrong given that ioctl is still defined (and I can use it in C just fine), but I can't find any architecture-specific hooks within swiftpm.